src/library/columncache.cpp

```diff
@@ -90,3 +93,100 @@ void ColumnCache::setColumns(const QStringList& columns) {
     m_columnSortByIndex.insert(m_columnIndexByEnum[COLUMN_PLAYLISTTRACKSTABLE_ARTIST], sortNoCase);
     m_columnSortByIndex.insert(m_columnIndexByEnum[COLUMN_PLAYLISTTRACKSTABLE_TITLE], sortNoCase);
 }
+
+void ColumnCache::slotSetKeySortOrder(double notation) {
```

Comment:
Now I got it: 
https://github.com/mixxxdj/mixxx/blob/94e9c0c93d985119ffb6403d774882c02c4ce6a4/src/track/keyutils.cpp#L125

DEFAULT is uses for CUSTOM. 

So I think CUSTOM should become 0 and we actually need the slotSetKeySortOrder() during the constructor.